include/OpenColorIO/OpenColorIO.h

```diff
@@ -186,53 +185,49 @@ extern OCIOEXPORT void UnsetEnvVariable(const char * name);
 //!cpp:function::
 extern OCIOEXPORT bool IsEnvVariablePresent(const char * name);
 
-///////////////////////////////////////////////////////////////////////////
-// Config
-
-// TODO: Move to .rst
-//
-// A config defines all the color spaces to be available at runtime.
-//
-// The color configuration (:cpp:class:`Config`) is the main object for
-// interacting with this library. It encapsulates all of the information
-// necessary to use customized :cpp:class:`ColorSpaceTransform` and
-// :cpp:class:`DisplayViewTransform` operations.
-//
-// See the :ref:`user-guide` for more information on
-// selecting, creating, and working with custom color configurations.
-//
-// For applications interested in using only one color config at
-// a time (this is the vast majority of apps), their API would
-// traditionally get the global configuration and use that, as opposed to
-// creating a new one. This simplifies the use case for
-// plugins and bindings, as it alleviates the need to pass around configuration
-// handles.
-//
-// An example of an application where this would not be sufficient would be
-// a multi-threaded image proxy server (daemon), which wished to handle
-// multiple show configurations in a single process concurrently. This
-// app would need to keep multiple configurations alive, and to manage them
-// appropriately.
-//
-// Roughly speaking, a novice user should select a
-// default configuration that most closely approximates the use case
-// (animation, visual effects, etc.), and set the :envvar:`OCIO` environment
-// variable to point at the root of that configuration.
-//
-// \note
-//    Initialization using environment variables is typically preferable in
-//    a multi-app ecosystem, as it allows all applications to be
-//    consistently configured.
-//
-// See :ref:`developers-usageexamples`
-
 /// Get the current configuration.
 extern OCIOEXPORT ConstConfigRcPtr GetCurrentConfig();
 
 /// Set the current configuration. This will then store a copy of the specified config.
 extern OCIOEXPORT void SetCurrentConfig(const ConstConfigRcPtr & config);
 
-
+/**
+ * \brief
+ * A config defines all the color spaces to be available at runtime.
+ *
+ * The color configuration (Config) is the main object for
+ * interacting with this library. It encapsulates all of the information
+ * necessary to use customized ColorSpaceTransform and
+ * DisplayViewTransform operations.
+ *
+ * See the \ref user-guide for more information on
+ * selecting, creating, and working with custom color configurations.
+ *
+ * For applications interested in using only one color config at
+ * a time (this is the vast majority of apps), their API would
+ * traditionally get the global configuration and use that, as opposed to
+ * creating a new one. This simplifies the use case for
```

Comment:
Would be nice to rewrap places where there are huge difference in adjacent lines.

Reply:
Good call. It looks like you guys are using roughly ~100 characters for documentation in other headers; I'll try to straighten things up to match that.

There are some documentation style inconsistencies throughout as well that I was gonna ask about; but first, I wanted to get to a point where everything was rendering properly. (My work on the docs has been a little haphazard, as I haven't figured out a great workflow for myself yet, and I don't totally know what I'm doing -- just rebuilding and trying to decipher error messages as they pop up. It's been a learning experience...)